test/contract_tests/test_ci_test.py

```diff
@@ -4,79 +4,81 @@
 from exasol_script_languages_container_ci.lib.ci_test import DBTestRunnerProtocol
 
 
-@pytest.fixture()
-def intact_flavor_path(flavors_path):
-    return str(flavors_path / "intact")
+class SuccessfulFlavorContract:
 
+    @pytest.fixture()
+    def flavor_path(self, flavors_path):
+        return str(flavors_path / "successful")
 
-@pytest.fixture()
-def broken_test_flavor_path(flavors_path):
-    return str(flavors_path / "broken-test")
+    @pytest.fixture()
+    def test_container(self, test_containers_folder):
+        return str(test_containers_folder / "successful")
 
 
-class IntactFlavorDBTestsContract:
+class SuccessfulFlavorDBTestsContract(SuccessfulFlavorContract):
 
     @pytest.fixture
     def db_test_runner(self) -> DBTestRunnerProtocol:
         raise NotImplementedError()
 
-    def test(self, db_test_runner, successful_test_test_container, intact_flavor_path):
-        try:
-            result = db_test_runner.run(flavor_path=(intact_flavor_path,), test_folder=(),
-                                        release_goal=('release',), workers=7,
-                                        docker_username=None, docker_password=None,
-                                        test_container_folder=successful_test_test_container)
-            assert result.tests_are_ok and result.command_line_output_path.exists()
-        except TaskRuntimeError as e:
-            for i in e.inner:
-                print(i)
-            raise e
```

Comment:
👍  Cool that we can drop the `except TaskRuntimeError ` now!

Reply:
this was only for debugging while developed, I will fix the root cause in the itde and then release